src/librustc_codegen_llvm/llvm/mod.rs

```diff
@@ -106,9 +106,9 @@ pub fn UnsetComdat(val: &'a Value) {
     }
 }
 
-pub fn SetUnnamedAddr(global: &'a Value, unnamed: bool) {
+pub fn SetUnnamedAddress(global: &'a Value, unnamed: UnnamedAddr) {
     unsafe {
-        LLVMSetUnnamedAddr(global, unnamed as Bool);
+        LLVMSetUnnamedAddress(global, unnamed);
     }
 }
```

Comment:
A bit confused about all of these wrappers, especially the ones that don't have any casts in them.

Reply:
It would be good to go through all these wrappers and remove those that don't carry their weight, but IMO that is a clean-up beyond the scope of this PR.